src/modules/media/queries/get-related-objects.graphql

```diff
@@ -0,0 +1,77 @@
+query getRelatedObjects($schemaIdentifier: String!, $meemooIdentifier: String!) {
+	object_ie(
+		where: {
+			schema_identifier: { _neq: $schemaIdentifier }
+			meemoo_identifier: { _eq: $meemooIdentifier }
```

Comment:
should also check that the items are inside the same maintainer:
![image](https://user-images.githubusercontent.com/1710840/161086543-994e02cc-8be5-4e8d-b225-dae5b4ff686e.png)

Reply:
fixed